README.md

```diff
@@ -69,26 +69,21 @@ On ArchLinux:
 sudo pacman -S  base-devel rsync opam pandoc jre8-openjdk mpfr maven
 ```
 
-On OSX, using [Homebrew](https://brew.sh/):
+On OSX, using [Homebrew](https://brew.sh/), after installing XCode:
```

Comment:
I'm not sure you need to install XCode. You need just need the command line tools package, which most users would already have.

Reply:
I will update this to mention the command line tools explicitly.